src/cljs/athens/parse_renderer.cljs

```diff
@@ -83,8 +83,8 @@
   (insta/transform
     {:block         (fn [& contents]
                       (concat [:span {:class "block" :style {:white-space "pre-line"}}] contents))
-     :component     (fn [contents]
-                      (components-util/render-component contents uid))
+     :component     (fn [& contents]
+                      (components-util/render-component (first contents) uid))
```

Comment:
Why are we making `& contents` into a list and then calling `first` on it?

Are arguments variadic? If they are, we need comments explaining why we're dropping the rest.

Reply:
This goes for `page-link` as well. Why do we sometimes use `&` and sometimes not?

Reply:
Arguments are variadic because the contents list's first element is the unparsed string (for more efficient matching across many regexes), while the rest are parsed tree for things like dynamic references & auto page creation. Will add documentation

Reply:
Docs added

Reply:
Thank you

Reply:
`page-link` might also need a bit more clarification - will add docs in a separate commit since it's unrelated to this.